ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1024,9 +1025,9 @@ static PrjFS_Result HandleFileNotification(
         notificationType,
         nullptr /* destinationRelativePath */);
     
-    if (partialFile && PrjFS_NotificationType_FileModified == notificationType)
+    if (result == 0 && placeholderFile && PrjFS_NotificationType_PreConvertToFull == notificationType)
     {
-        // PrjFS_NotificationType_FileModified is a post-modified FileOp event (that cannot be stopped
+        // PrjFS_NotificationType_PreConvertToFull is a event (that cannot be stopped
         // by the provider) and so there's no need to check the result of the call to NotifyOperation
```

Comment:
This comment is out of date:

```
// PrjFS_NotificationType_PreConvertToFull is a event (that cannot be stopped
// by the provider) and so there's no need to check the result of the call to NotifyOperation
```

Now the provider can stop the event (and we are checking the result of `NotifyOperation`